### DISCUSSION TITLE
Tier Zero - How To Reach Tier Two By Next Tuesday [SPOILERS]

### DISCUSSION THREAD

**Discussion by Tangle10** [endless-sky/endless-sky#10329] [Lore]
### or, An Accounting Of The Disparity In Starting Positions

Apologies for the half-flippant, half-grandiose title. My purpose with this discussion is to provide a place for semi-long-term talk about the milestones various species reached _before_ they invented the hyperdrive, as a point of comparison for other new species. I posted this on the ESC discord earlier, but of course anything posted there is fairly quickly pushed offscreen and forgotten.

So, here are the species already in the game- arranged in rough chronological order.

Drak - We have no idea.
Pug - We have no idea.
Wanderers - We have no idea. They may have been uplifted to spaceflight by the Pug.
Quarg - We have almost no idea. Their homeworld likely being a small moon with thin air probably makes things easier.
Builders - We have almost no idea. It's stated in "Beneath the Shell" that the Ka'sei went beyond the hyperlanes in a way very few spacecraft ever did, so they likely did not use generation ships.
Hai - We have very little idea. Mirrorlake has cities at least intended to mimic ancient ones on them, so perhaps the cities were originally built before the hyperdrive.
Sheragi - We have basically no idea. Like the Builders, sublight interstellar capacity seems to not have been a priority.
Bunrodea - At least one interstellar generation ship left for another star, and the two worlds developed separately for millennia- one developing hyperdrive but for unknown reasons not contacting the other, the other building a return generation ship.
Korath - long-range interplanetary, but not interstellar, exploration at high speed.
Coalition/Heliarchs - three separate instances of space- and star-faring. Unsure of extent of each, generally unstated?
Humanity - a "fleet" of generation ships taking an indeterminate time to reach Alpha Centauri, plus cities on Luna and Mars. 
Remnant - started with hyperdrive, immediately managed to place themselves in the lap of three separate starfaring species' technology.
Gegno - many interstellar generation ships, but apparently generally limited to a few nearby destinations.

Now, the species that exist in plugin-to-be-made-into-pull-request or pull request form that have been active relatively recently.

Xapleaux - not stated
Successors - interstellar generation ships launched, but not arrived before drive designed?
Naltok -interstellar generation ship, separate development for ~300 years
Bashkmul - interstellar ships launched, took hyperdrive for Rigra to actually colonize; Aert had no starfaring prior to HD?
Avgi - extreme interplanetary development. multiple waves of high-speed (0.25-0.4c) interstellar ships, nearest ~7 worlds colonized and terraformation started centuries prior to HD development. 
Incipias - thriving single-system civilization currently experimenting with hyperdrive

And I forgot the Searchers at first, so in apology I will post mOctave's summation of the situation for the Searchers in its own section.

Searchers - Them trying to cram something like 20 billion people on one small planet and failing miserably, paired with sighting aliens and therefore knowing something like a hyperdrive was possible, probably intentionally killing off billions of their people to stay alive, and then finally having scientific breakthroughs. No interstellar generation ships.

**Comment by Saugia** [comment]
There are no Rulei in Ba Sing Se.

**Reply by Tangle10** [reply]
The fact that I'm tired enough that I don't remember if I wrote anything for the Rulei and you just edited it away or if I just didn't makes this _extra spooky_.

**Reply by Saugia** [reply]
Haha; I didn't delete anything. I've said that the Rulei were unique and discovered the JD first, not the HD, as in the past we said nothing explicitly states that can't happen (and who knows, that could have happened with the Drak or someone too), so they'd also be "We have no idea".

Or keep it mysterious and don't add them. ~~If you do, I'll delete these comments.~~

**Reply by Tangle10** [reply]
> Haha; I didn't delete anything. I've said that the Rulei were unique and discovered the JD first, not the HD, as in the past we said nothing explicitly states that can't happen (and who knows, that could have happened with the Drak or someone too), so they'd also be "We have no idea".
> 
> Or keep it mysterious and don't add them. ~If you do, I'll delete these comments.~

Ah, I see! It's really funny, because I remember writing down "Rulei -" at SOME point. I wonder where I did it....

**Comment by Ferociousfeind** [comment]
The new post that has all vanilla aliens _and_ all prominent to-be-vanilla PR or plugin aliens, but doesn't have mine...
I've really fallen off. Super taken the L. Cringe. Pwned. Losing.

I should probably resume work on writing Erelinn story, and maybe brute-force cleaning up the Erelinn models

**Reply by Tangle10** [reply]
My apologies. Where were they at before the hyperdrive?

**Reply by Ferociousfeind** [reply]
Well, they're where you find them during the game- they haven't developed a hyperdrive yet, and they're still only interplanetary! The territorial ants are embroiled in a cold interstellar war between two major hives, and the big idea is that you accidentally (or purposefully?) introduce hyperdrive technology to them, and your favoritism helps the cold war go hot.

As for actual technology, they're mainly inspired by the technologies of CoaDE, so it's all near-future and proven tech, plus just a touch of magic stuff to make it mostly fit with ES. They haven't made generation ships yet (politics on a live-crew ship would get complicated fast, and a seed ship- Azure below called it a sleeper-ship- would be logistically intense and impractical. So, they haven't colonized nearby star systems, they're stuck in the one system. Within this system, though, they also have similar technologies to the Avgi, although with some personal-taste modifications.

Erelinn ships are mostly designed for interplanetary combat, with ferocious engines and bulky fuel tanks, and bloodthirsty lasers, cannons, and guided missiles. Armor is largely plates or cast metal, whatever eats railgun shots the best, while shield technology is virtually nonexistent (I consider it too magical, like the hyperdrive, for them to develop it). They've unlocked fission and fusion for power generation, and miniaturization to make fusion reactors almost viable for the most massive warships! Their engines all use fuel, and they have fuel pods to offset the costs of moving and orienting and powering those hungry reactors. Their engines are also in a variety of flavors, with ionized plasma, good old hydrocarbons, and the power-hungry nuclear option.

Okay but, actually as for, like, technologies developed that are _relevant to what you've already documented._ Like, historical timeline and general accomplishments rather than moments-before-the-hyperdrive state of society...

Much of their pre-history was spent under the oppression of natural predators- giant birds and stuff that like to eat ants. In fact, it's not even the Erelinn that clawed the Erelinn out of the evolutionary struggle for life and into the development of society... Once they were enlightened, society developed rapidly due to their aggressive, territorial nature. Hundreds of hives collapsed to dozens, then only a few massive globe-spanning colonies, all underground to stay hidden from predators.

Erelinn history begins with the uplifting by the mysterious glowworms, and then the discovery of fire, and metalworking, and steam power, and rapid excavation, and firearms, and dominance over their natural predators, and industrialization, and slavery and subjugation, and geothermal power and political struggle, and nuclear isotopes, and heavier-than-air flight, and aircraft interceptors, and the idea of kamikaze (worker and soldier ant lives do not matter very much to the queen, it is like losing a finger (that will grow back) to blow off the arm or your opponent), and ever-higher-flying craft to avoid interception, until the thinning atmosphere at higher elevations force rockets to develop.

If I had to give an estimate, the Erelinn discovered fire at 0, metalworking at 500, steam power at 800, firearms (and thus dominance over their natural predators) at 1000, nuclear isotopes at 1200, and heavier-than-air flying machines at 1500. The arms race in the sky brought about rockets a mere 30 years after the first flying machines were used for war. If the player finds them 200 years after they discover spaceflight, after the slight plateau after the space tech boom (so 1730 years after fire), then the Erelinn discover fire in the year 1284 AD, and although they have a late start compared to humans, they rapidly speed through technology until the modern (ES) day.

With access to the skies (but slowly, through space), Erelinn infighting slowed down until the chain reactions came to a halt, like a nuclear isotope slowly becoming subcritical. The violent and inhumane fighting turned from physical combat to political opposition, and the queens of the Erelinn look to the skies for new growth potential. The dominant queen cheerily sends a subservient "princess" and her hive of workers to the nearly mostly habitable planet somewhat closer to their parent star- an arid world, but with enough water to live off of.

As you might expect with such aggressive, territorial creatures, over time this colony becomes discontent with the remote and not-backed-up-by-big-guns iron fist of the dominant homeworld queen, and the colony princess becomes defiant enough to declare independence. This leads straight to the modern day, where the player finds them. (It was also a biological inevitability- the colony's princess felt no challenge and developed into a fully mature queen.)

I can, uh, ramble a whole lot more, if you want!! If anyone wants, really!! I really really like these and I've even restrained myself just a little on a couple of these paragraphs.

**Reply by Ferociousfeind** [reply]
If I HAD to condense it into one sentence for the state of space technology "when they develop the hyperdrive", it'd be...

> Xapleaux - not stated
> Successors - interstellar generation ships launched, but not arrived before drive designed?
> Naltok -interstellar generation ship, separate development for ~300 years
> Bashkmul - interstellar ships launched, took hyperdrive for Rigra to actually colonize; Aert had no starfaring prior to HD?
> Avgi - extreme interplanetary development. multiple waves of high-speed (0.25-0.4c) interstellar ships, nearest ~7 worlds colonized and terraformation started centuries prior to HD development.
> Incipias - thriving single-system civilization currently experimenting with hyperdrive
> Erelinn - Petty infighting stalling outward expansionary tech development. In-system advanced colonization, no generation ships

**Reply by Tangle10** [reply]
> If I HAD to condense it into one sentence for the state of space technology "when they develop the hyperdrive", it'd be...
> 
> > Xapleaux - not stated
> > Successors - interstellar generation ships launched, but not arrived before drive designed?
> > Naltok -interstellar generation ship, separate development for ~300 years
> > Bashkmul - interstellar ships launched, took hyperdrive for Rigra to actually colonize; Aert had no starfaring prior to HD?
> > Avgi - extreme interplanetary development. multiple waves of high-speed (0.25-0.4c) interstellar ships, nearest ~7 worlds colonized and terraformation started centuries prior to HD development.
> > Incipias - thriving single-system civilization currently experimenting with hyperdrive
> > Erelinn - Petty infighting stalling outward expansionary tech development. In-system advanced colonization, no generation ships

Very, very cool- thank you for elaborating! I do quite like the idea of another pre-hyperdrive species. Makes the galaxy feel more even, and as a bonus they don't need as many systems allotted to 'em (plus the systems that _are_ can be very fun to explore).

**Reply by Ferociousfeind** [reply]
Can confirm, I've put a lot of love into the home system, and I put special thought into the 18 systems I made to be hyperlane-connected to their homeworld. Draan, their home system, has the inhabited planets "Cael" (home) and "Au Se'el" (belonging to Se'el), and the colonized moons Aketso (Au Cael), Ni'ali (Au Se'el), Faliel (Au Se'el), Keshi (Au Cael), and Ikanatsu (Au Sonefta).

Outside, they have received CONSIDERATION, but not necessarily LOVE, y'know? Interstellar civilization is further along in the campaign, whenever that gets written, so we don't need those planets all outlined right now.

Also, I have a bunch of flagship style variants, with erelinn faction flags on them, but they won't actually show up :( so eventually I also have to figure out what's up with that, why variants with alternate sprites doesn't seem to be working

**Comment by Azure3141** [comment]
> Avgi - extreme interplanetary development. multiple waves of high-speed (0.25-0.4c) interstellar ships, nearest ~7 worlds colonized and terraformation started centuries prior to HD development.

I'll note that the Avgi only colonized the three nearest systems, Clip, Soar, and Thread, with an attempt in Span that failed. I imagine the terraforming needs in the first three systems were pretty light though, with Livolua and Stilacrest being almost ideal for Avgi settlement and Cyreal being fairly nice (aside from its low gravity). This may or may not have been set up by a more advanced species, but only for those worlds. Most of the other worlds you can find in Avgi space are less hospitable and would likely need terraforming in the future, with a few exceptions.

Avgi starships themselves weren't generation ships, but rather sleeper ships with colonists held in cryostasis. Trips to the nearest stars (only a few lightyears away) would be measured in the low decades to below a single decade just before the invention of the hyperdrive, so the cryostasis would largely be to preserve lifespan and allow for tens of thousands of Avgi to be shipped at a time, unfrozen as the ability to support them was built.

I imagine there were some very early ships launched with some form of mag-Orion drive at a few % of c, taking centuries to arrive. This would have happened well within the first century of Avgi space exploration, which of course started off with a bang and used Orion drive ships from the beginning to very rapidly colonize most of the Anax system. These missions would have had a low success rate due to the length of the journey involved, and all would be overtaken by faster starships launched decades or sometimes a century later. There might even be one or more early Orion ships still in transit to more distant targets, (5-10 lightyears) which at 0.02c would make them arrive at approximately the present day 👀

Fusion drives would take a while longer to be developed, probably at least another century for ones suitable for interstellar travel (mostly laser/particle beam ICF and maybe z-pinches earlier on). These starships would be capable of achieving speeds of 0.1c initially, steadily climbing as drives improved. They would find themselves quickly competing with beamed propulsion methods as giant lasers and particle beam stations were built around Anax, which would be able to achieve similar velocities (and would often be used in tandem). Plasmadynes for braking (and eventually accelerating through dynamic soaring and shear-pellet sailing) would also be developed within this timespan, but would need an initial acceleration phase using either drives or beams to work effectively.

The most recent, and last major development in STL interstellar travel would be the introduction of antimatter propulsion. While the drives themselves would be much simpler and easier to design than the most advanced fusion torches, actually producing the quantities of antimatter necessary to fuel them would prove much more of a challenge, largely one of scale. Accelerators optimized for antimatter production would be built en-masse using material from Omanet, most notably Antivili Station, with total production reaching perhaps as much as a few tons per year. Starships equipped with a beam-core antimatter drive could reach cruising speeds of 0.4c, with room for improvement had the hyperdrive not been invented only a few decades afterward.

The hyperdrive would crater the antimatter economy, as while fusion torches were useful enough for intrasystem travel to survive, antimatter itself was too scarce to use for most ships, especially as they spread away from Anax and could easily refuel with local deuterium found in water practically everywhere. I imagine a few very rich organizations would continue to use other types of antimatter drives like plasma cores within the Anax system due to their vastly simpler operation and ability to be miniaturized, largely due to the depressed prices as the demand for interstellar antimatter collapsed and a production glut flooded the market. However, due to production being centralized almost entirely within Anax, and accelerators being fairly complex to produce in other systems, antimatter drives would not become widespread within the winder interstellar Avgi civilization. And with the Incursion either destroying or causing the abandonment of the antimatter farms around Anax, antimatter production is unlikely to return to previous levels in the near future.

The modern technological state of Avgi propulsion is therefore somewhat determined more by practicality and convenience than what is the most "advanced" option. Fusion drives have been optimized over centuries, work very well, and run on very cheap fuel (deuterium), so there is not really a reason to switch away from them.

**Reply by Tangle10** [reply]
Ah, noted! I was pretty sure the Avgi didn't use generation ships, but I had thought the other three systems they currently inhabit were all pre-Hyperdrive.

**Reply by Azure3141** [reply]
A generation ship would require perhaps more social cohesion than the Avgi would be capable of (only half joking here)

**Reply by Tangle10** [reply]
> A generation ship would require perhaps more social cohesion than the Avgi would be capable of (only half joking here)

Honestly, a generation ship is a fascinating half-analogue to the generations-long(?) siege the Avgi are currently under... hmm.
